ajax/libs/iris-ws/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "iris-ws",
+  "title": "iris-ws",
+  "filename": "client.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kataras/iris.git",
+    "fileMap": [
+      {
+        "basePath": "websocket",
+        "files": [
+          "client.js",
+          "client.min.js"
```

Comment:
Let's use glob pattern `client*.+(js|map)` to replace line 12-13.

Reply:
done